README.md

```diff
@@ -30,11 +30,18 @@ Run `npx task --help` to see all available commands and their description.
 * use sequelize-cli local to generate migrations (because of timestamp)
 
 
+## Documentation
+
+The documentation can be accessed on `app-url:docs-port/docs-url`
```

Comment:
Put example here  -> i.e. `http://localhost:8080/api/swagger/` as these don't look like variables really

Reply:
Wait something is wrong here. put here ${ENV_VARIABLE} like ${SWAGGER_DOCS_URL}